default_metadata/component/checkboxes.json

```
@@ -0,0 +1,9 @@
+{
+  "_id": "definition.checkboxes",
+  "_type": "checkbox",
+  "errors": {},
+  "hint": "Component hint",
+  "items": [],
+  "name": "component-name",
+  "legend": "Required legend"
```

```suggestion
  "legend": "Question"
```

again, i think that's what's in the prototype